recipes/dcmtk/all/conanfile.py

```diff
@@ -42,6 +45,10 @@ class DCMTKConan(ConanFile):
         "with_libsndfile": "deprecated",
         "with_libtiff": True,
         "with_tcpwrappers": False,
+        "builtin_private_tags": False,
+        "external_dictionary": True,
+        "wide_io": False,
+        "libxml2:shared": False
```

Comment:
I think the default of `libxml2` is already `shared=False`.
No recipe in this repo is changing the defaults option values of its requirements.
```suggestion
```